pyg_lib/testing.py

```diff
@@ -27,14 +27,16 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def withDataset(group: str, name: str) -> Callable:
+def withDataset(group: str, name: str,
+                return_csc: Optional[bool] = False) -> Callable:
```

Comment:
I don't think we need this option. We can call both samplers just with the returned CSR type. It will result in the same computation.

Reply:
Good idea, thanks.

Reply:
Done.